bodhi/server/services/graphql.py

```diff
@@ -48,11 +48,107 @@ class Query(graphene.ObjectType):
     """Allow querying objects."""
 
     allReleases = graphene.List(Release)
+    specificReleases = graphene.Field(lambda: graphene.List(Release), 
+        name=graphene.String(), long_name=graphene.String(),
+        version=graphene.String(), id_prefix=graphene.String(),
+        branch=graphene.String(), dist_tag=graphene.String(),
+        stable_tag=graphene.String(), testing_tag=graphene.String(),
+        candidate_tag=graphene.String(), pending_signing_tag=graphene.String(),
+        pending_testing_tag=graphene.String(), pending_stable_tag=graphene.String(),
+        override_tag=graphene.String(), mail_template=graphene.String(),
+        id=graphene.Int(), composed_by_bodhi=graphene.Boolean(),
+        create_automatic_updates=graphene.Boolean(), testing_repository=graphene.String(),
+        state=graphene.String(), package_manager=graphene.String())
 
     def resolve_allReleases(self, info):
         """Answer Queries by fetching data from the Schema."""
         query = Release.get_query(info)  # SQLAlchemy query
         return query.all()
 
+    def resolve_specificReleases(self, info, **args):
+        """Resolving releases with a given argument"""
+        query = Release.get_query(info)
+
+        id_prefix = args.get("id_prefix")
```

Comment:
same here we can really focus on the following fields `name, id_prefix, state, composed_by_bodhi`